service/grails-app/views/patronRequest/_patronRequest.gson

```diff
@@ -63,8 +64,9 @@ final def toExclude = fullRecord ? [
 
 json g.render(patronRequest, [expand: toExpand,
                               excludes: toExclude]) {
+
   if (fullRecord) {
-    validActions patronRequest.validActions
+    validActions g.render(patronRequest.validActions, [excludes: ['actionEvent', 'resultList']])
```

Comment:
Might be better to just list what we want to include, perhaps just the code and the new booleans to start?